File: backend-web/commons/src/main/java/com/it/marathon/dto/ReportDTO.java
package com.it.marathon.dto;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class ReportDTO {

    private String imageBase64;
    private Categories category;
    private String comments;
    private Contact contacts;
    private String geoTagging;
    private String timestamp;
}

File: CommunityCleanAndroidApp/CommunityClean/app/src/main/java/com/example/communityclean/dtos/ReportDto.kt
package com.example.communityclean.dtos

data class ReportDto(
    val imageBase64: String,
    val category: Int,
    val comments: String?,
    val contacts: Contacts?,
    val geoTagging: GeoTag,
    val timestamp: String
)

data class Contacts(
    val email: String,
    val tel: String,
    val firstName: String,
    val lastName: String
)

data class GeoTag(
    val latitude: Double,
    val longitude: Double
)
